domains/starshine.json

```diff
@@ -0,0 +1,12 @@
+{
+        "owner": {
+           "username": "thaddeuskkr",
+           "email": "",
+           "discord": "275830234262142978",
+           "OWL": "eyJlbmMiOiJBMTI4Q0JDLUhTMjU2IiwiYWxnIjoiUlNBLU9BRVAiLCJraWQiOiJaa1VsRmRqVThiUEstLXVVM2JJR09PVHFYYVFFS1ZINFVXOW53MTR6WTJnIn0.lLFfIW3EiPjgK-i-I5E56ERBvjl-ueM91PRyYM6wRClZzuoAj4nQyqL57dSZjhO_4Hbu4ZubbM1I_NI8Vt4_SlsFmxgL-hrvH9tf7rv-hX-TZtBWT_LQQUmhd1ZViGwELTly797M8VXZ39ER2gDmzruxjh8eQEHBCeJujWuUNicigehUgfk-Pk9rSchVlR3TUcwL-wylY58KvBLQwIeMmqUW2qs_E_iKCwk_8FQH-IUb_4W58r6wBHh6MqiP72Py9ssDbxrV59e1eGrQZXYJJM1rUZHoHzW1KX84FgT0QCGc1cSiV-Sdc3p6WceRY6Tf1akfTNjuga61kpgzluoTxg.QwFk8iYMpzefwXgErTr9cQ.kwMwAIrFDxCKfvSxx3Jpvb6ZkiFcp56BFoBg7KQMsYw4PBK60JNzjpMpgMqZ7jymnYJFTz5img3xLlxBF7UuUy_Z9MgHCHaVR3W7970smoQ.aBJoeQFaHiIyXkpQa1Gx7g"
+        },
+    
+        "record": {
+            "A": ["138.2.70.1"]
```

Comment:
Access denied, please give a preview of your website.

Reply:
oh sorry, i can't have caddy reverse proxy right now cause it tries to obtain an ssl cert and gets ratelimited after failing the challenge.

here's the preview for what both domains are supposed to show:
![image](https://github.com/user-attachments/assets/5e88ced3-f448-43ba-bb15-dc071349da72)

lmk if anything else needed

Reply:
Ye thats fine, just needed the sites preview, will merge the PR now